src/styles/blocks/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 33px 16px 16px;
```

```suggestion
  flex-wrap: wrap;
  padding: 33px 16px 16px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```
check [markup](https://www.figma.com/file/ojkArVazq7vsX0nbpn9CxZ/Moyo-%2F-Catalog-(ENG)?type=design&node-id=32249-772&mode=design&t=hViVqElPes9LYUcA-0) and add border for card. it should help to fix tests.